colcon_bundle/verb/_overlay_utilities.py

```diff
@@ -86,31 +94,46 @@ def create_dependencies_overlay(staging_path, overlay_path):
         dependencies_tar_gz_path
     ))
 
-    _generate_template(
-        'v2_setup.jinja2.sh',
-        'v2_setup.bash',
-        _CONTEXT_VAR_BASH
+    assets_directory = os.path.join(
+        os.path.dirname(os.path.realpath(__file__)),
+        'assets'
+    )
+
+    shellscript_path = os.path.join(
```

Comment:
instead of defining all these variables, which blend together and become basically unreadable, I think this whole function would read a lot easier as just a sequence of copies and renders, e.g.:

```
assets = pathlib.Path(__file__).parent.absolute() / 'assets'
stage = dependencies_staging_path
shutil.copy2(assets / 'v1_setup.sh', stage / 'v1_setup.sh')
render_template(assets / 'v2_setup.sh.j2', stage / 'v2_setup.sh', CONTEXT_VAR_SH)
# etc... ends up being like 10 lines total
```